README.md

```diff
@@ -1,6 +1,6 @@
 # LitmusPaper
 
-Backend health tester for HA Services, or as an agent-check for HAProxy
+LitmusPaper is a backend health tester for HA Services.
```

Comment:
Style nit:` s/HA/Highly Available (HA)/` for this first intro of the abbreviation. It improves searchability a tad, and also clarifies the acronym for those who might not use it (which is probably pretty rare, but might happen).